### PR TITLE
API for ssn id: TSHttpSsnIdGet

### PR DESCRIPTION
**Pull request by dyrock**

### CODE REVIEW

**Comment by zwoop** [comment]
[approve ci].

File: doc/developer-guide/api/functions/TSHttpSsnIdGet.en.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Description
+===========
+
+:func:`TSHttpSsnIdGet` returns the unique identifier for this client session.
+
+Return Values
+=============
+
+Returns a :type:`int` as the unique ID
```
Comment (SolidWallOfCode) [inline doc/developer-guide/api/functions/TSHttpSsnIdGet.en.rst]: Why is this `int` if the declared return type is `int64_t`?

Comment (dyrock) [reply doc/developer-guide/api/functions/TSHttpSsnIdGet.en.rst]: oops

**Comment by zwoop** [comment]
Don't merge PRs without updating Milestone / Labels / Assignee.

**Pull request merged**